ui/components/app/edit-gas-display/edit-gas-display.component.js

```diff
@@ -156,6 +148,16 @@ export default function EditGasDisplay({
             />
           </div>
         )}
+        {isNetworkBusy ? (
```

Comment:
This warning applies to the EIP-1559 V1 UI. Note that this warning appears within the edit gas modal, NOT the confirm transaction screen, unlike EIP-1559 V2. I could have added it to the confirm transaction screen, but it would be more work. Given that we are soon transitioning to the V2 UI I feel like this is an okay compromise for now, but happy to change if not.

Reply:
Looks good, in V@ as I understand we are showing network busy only on edit gas fee modal.

Reply:
In V2 it's slightly different — we are showing the network busy message on the confirm transaction screen, but not in the edit gas fee modal.